packages/amazon-ecs-cni-plugins/amazon-ecs-cni-plugins.spec

```diff
@@ -1,11 +1,13 @@
 %global ecscni_goproject github.com/aws
 %global ecscni_gorepo amazon-ecs-cni-plugins
 %global ecscni_goimport %{ecscni_goproject}/%{ecscni_gorepo}
-%global ecscni_gitrev 53a8481891251e66e35847554d52a13fc7c4fd03
+%global ecscni_gitrev 7b4ec6016ab221469fa3abfc00ea7c05f236c26c
```

Comment:
We should keep this aligned with [the `ecs-agent` submodule](https://github.com/aws/amazon-ecs-agent).

Reply:
Good point, I see ecs-agent is still using / testing against `2020.09.0`  `53a8481891251e66e35847554d52a13fc7c4fd03` https://github.com/aws/amazon-ecs-agent/blob/a08050481d214746d5e2c6f73ad360c69f63cd95/agent/ecscni/plugin_test.go#L31-L32

I'll slim this PR down to just the versioning change